src/App.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
-import logo from './logo.svg'
+// import logo from './logo.svg'
 import './App.css'
+import Home from './pages/Home'
 
 function App() {
   return (
     <div className='App'>
-      <header className='App-header'>
+      {/* <header className='App-header'>
```

Comment:
Can you please remove the commented code. We will not need it later so need to keep it as comments.

Reply:
Yeah Sure !!